frontend/public/components/utils/request-size-input.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-undef, no-unused-vars */
+import * as React from 'react';
+import { Dropdown } from '.';
+
+export class RequestSizeInput extends React.Component<RequestSizeInputProps> {
+
+  onChange: React.ReactEventHandler<HTMLInputElement> = event => {
+    this.props.onChange(event);
+  };
+
+  onDropdownChange = event => {
+    let obj = { currentTarget: { name: `${this.props.name}Unit`, value: event } };
+    this.props.onChange(obj);
+  }
+
+  render() {
+    const inputName = `${this.props.name}Value`;
+    const dropdownName = `${this.props.name}Unit`;
+    return (
+      <div className="form-group">
+        <div className="input-group">
+          <input
+            className="form-control"
+            type="number"
+            onChange={this.onChange}
+            placeholder={this.props.placeholder}
+            aria-describedby={this.props.ariaName}
+            name={inputName}
+            required={this.props.required}
+          />
+          <Dropdown
+            title={this.props.defaultRequestSizeUnit}
+            selectedKey={this.props.defaultRequestSizeUnit}
+            name={dropdownName}
+            className="btn-group"
+            items={this.props.dropdownUnits}
+            onChange={this.onDropdownChange}
+            required={this.props.required}
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+export type RequestSizeInputProps = {
+  placeholder?: string;
+  name: string;
+  onChange: Function;
+  required?: boolean;
+  dropdownUnits: any;
+  defaultRequestSizeUnit: string;
+  ariaName?: string;
```

Comment:
Since there are [multiple aria attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/An_overview_of_accessible_web_applications_and_widgets), I would suggest to name this attribute `describedBy`.
I've already sent a PR which adding this attribute:
https://github.com/openshift/console/pull/672/files#diff-447b013466c334bed9fb476dc38fc194R293